src/route-matcher.ts

```diff
@@ -1,16 +1,16 @@
-// @ts-ignore
+// @ts-expect-error
 export function routeMatcher(paths) {
```

Comment:
```suggestion
export function routeMatcher(paths: string) {
```

With this simple addition, we can remove all the `// @ts-expect-error` in this file